### PR TITLE
Update feature list in README.md

### PR DESCRIPTION
**Pull request by paulwarren-wk**
#### Reason for change

List of features in README.md is out-of-date.

#### Description of change

Feature list updated.

Marking as draft, as I have some unresolved queries (see comments)

#### Steps to Test

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: README.md [hunk 1]
```diff
@@ -40,23 +40,34 @@ Need help with Arelle? Go check out [our documentation][read-the-docs].
...
+    - HMRC (UK)
+    - CIPC (South Africa)
+    - FERC (US Federal Energy Regulatory Commission)
+- Integrated support for [Arelle Inline XBRL Viewer][viewer].
+- Extensible plugin architecture.
+- Support for XF text-based Formula and XULE validation rules.
 - The Web Service API allows XBRL integration with applications, such as those in
   Excel, Java or Oracle.
+- Instance creation is supported using forms defined by the table linkbase
+  (Eurofiling version).
```
Comment (paulwarren-wk) [inline README.md:64]: I think that this is referring to the Acsone fork.  In any event, I think this is not a feature available in standard Arelle, and we shouldn't include it here.

File: README.md [hunk 2]
```diff
@@ -40,23 +40,34 @@ Need help with Arelle? Go check out [our documentation][read-the-docs].
...
+- RSS Watch facility.
```
Comment (paulwarren-wk) [inline README.md:65]: I'm not sure exactly what this is referring to.  I assume it's something that reads the SEC filing feed, but I think we should either expand or remove this point.

File: README.md [hunk 3]
```diff
@@ -40,23 +40,34 @@ Need help with Arelle? Go check out [our documentation][read-the-docs].
...
 - QuickBooks is supported by XBRL-GL.
```
Comment (paulwarren-wk) [inline README.md:66]: I'm not sure what this means.

File: README.md (hunk 1 above)
Comment (hermfischer) [reply README.md:64]: Arelle TLB (the old version) has forever allowed instance creation, Ascone developed that further from the old version and built a business.  Anyway it's not REC yet (on my plate).  But it DOES create instances.

File: README.md (hunk 2 above)
Comment (hermfischer) [reply README.md:65]: Only TLB instances can be created.  The wording is correct to that limitation.

Comment (austinmatherne-wk) [reply README.md:65]: I haven't used this, but it's in the tools menu (`Tools` -> `RSS Watch`) with a dropdown that contains different SEC feeds, validation options, alert options, and a start/stop function.

File: README.md (hunk 3 above)
Comment (austinmatherne-wk) [reply README.md:66]: It's referring [to this](https://github.com/Arelle/Arelle/blob/2c522ed0ffe57e273d6596dd38bf62568d2eb4d8/arelle/CntlrQuickBooks.py). We don't test to make sure it works. I'd vote dropping it from the readme.

File: README.md (hunk 1 above)
Comment (paulwarren-wk) [reply README.md:64]: What does "Eurofiling version" refer to?  Is this is one of the earlier drafts of the TLB spec?

File: README.md (hunk 3 above)
Comment (paulwarren-wk) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (paulwarren-wk) [reply README.md]: Just tried running it, and it looks like it was broken by c0677ecdc5501c6effc96c5d5eee4d0c25a362ab (#680)

I've expanded the description in 078b3171 and raised #853.

File: README.md (hunk 1 above)
Comment (paulwarren-wk) [reply README.md]: Thanks for the clarification - now done.

**Review by austinmatherne-wk — APPROVED** [review]

**Pull request merged**